Native image build error due to logging extension

Hello all,

I'm facing a serious issue concerning the use of `quarkus-google-cloud-logging` when building quarkus native image:

`org.graalvm.compiler.java.BytecodeParser$BytecodeParserError: com.oracle.graal.pointsto.constraints.UnresolvedElementException: Discovered unresolved method during parsing: io.grpc.alts.GoogleDefaultChannelCredentials$Builder.callCredentials(io.grpc.CallCredentials). This error is reported at image build time because class com.google.api.gax.grpc.InstantiatingGrpcChannelProvider is registered for linking at image build time by command line
`

Building the native image without this dependency works fine even with the existance of `quarkus-google-cloud-storage`.

`pom.xml` snippet:
```xml
<dependencyManagement>
    <dependencies>
      <dependency>
        <groupId>io.quarkus.platform</groupId>
        <artifactId>quarkus-bom</artifactId>
        <version>3.5.1</version>
        <type>pom</type>
        <scope>import</scope>
      </dependency>
      <dependency>
        <groupId>io.quarkiverse.googlecloudservices</groupId>
        <artifactId>quarkus-google-cloud-services-bom</artifactId>
        <version>2.6.0</version>
        <type>pom</type>
        <scope>import</scope>
      </dependency>
    </dependencies>
  </dependencyManagement>
...
<dependencies>
    <dependency>
        <groupId>io.quarkiverse.googlecloudservices</groupId>
        <artifactId>quarkus-google-cloud-logging</artifactId>
    </dependency>
    <dependency>
        <groupId>io.quarkiverse.googlecloudservices</groupId>
        <artifactId>quarkus-google-cloud-storage</artifactId>
    </dependency>
</dependencies>
```

Full logs:
```shell
Step #0 - "mavenBuild": [1/7] Initializing...                                                                                   (37.2s @ 0.54GB)
Step #0 - "mavenBuild":  Version info: 'GraalVM 22.3.0 Java 17 CE'
Step #0 - "mavenBuild":  Java version info: '17.0.5+8-jvmci-22.3-b08'
Step #0 - "mavenBuild":  C compiler: gcc (redhat, x86_64, 8.5.0)
Step #0 - "mavenBuild":  Garbage collector: Serial GC
Step #0 - "mavenBuild":  6 user-specific feature(s)
Step #0 - "mavenBuild":  - com.google.api.gax.grpc.nativeimage.GrpcNettyFeature
Step #0 - "mavenBuild":  - com.google.api.gax.nativeimage.GoogleJsonClientFeature
Step #0 - "mavenBuild":  - com.google.api.gax.nativeimage.OpenCensusFeature
Step #0 - "mavenBuild":  - com.oracle.svm.thirdparty.gson.GsonFeature
Step #0 - "mavenBuild":  - io.quarkus.runner.Feature: Auto-generated class by Quarkus from the existing extensions
Step #0 - "mavenBuild":  - io.quarkus.runtime.graal.DisableLoggingFeature: Disables INFO logging during the analysis phase
Step #0 - "mavenBuild": Warning: RecomputeFieldValue.FieldOffset automatic substitution failed. The automatic substitution registration was attempted because a call to sun.misc.Unsafe.objectFieldOffset(Field) was detected in the static initializer of com.google.protobuf.UnsafeUtil. Detailed failure reason(s): The argument of Unsafe.objectFieldOffset(Field) is not a constant field., Could not determine the field where the value produced by the call to sun.misc.Unsafe.objectFieldOffset(Field) for the field offset computation is stored. The call is not directly followed by a field store or by a sign extend node followed directly by a field store. 
Step #0 - "mavenBuild": [2/7] Performing analysis...  []                                                                       (141.4s @ 1.48GB)
Step #0 - "mavenBuild":   14,623 (93.61%) of 15,621 classes reachable
Step #0 - "mavenBuild":   20,564 (60.75%) of 33,851 fields reachable
Step #0 - "mavenBuild":   70,567 (84.52%) of 83,494 methods reachable
Step #0 - "mavenBuild":    1,286 classes,     0 fields, and     0 methods registered for reflection
Step #0 - "mavenBuild": 
Step #0 - "mavenBuild": Fatal error: com.oracle.graal.pointsto.util.AnalysisError$ParsingError: Error encountered while parsing com.google.api.gax.grpc.InstantiatingGrpcChannelProvider$$Lambda$eb484b81b2f172240bae44657dd3824463112836.createSingleChannel() 
Step #0 - "mavenBuild": Parsing context:
Step #0 - "mavenBuild":    at com.google.api.gax.grpc.InstantiatingGrpcChannelProvider$$Lambda$1988/0x00000007c28a6bb0.createSingleChannel(Unknown Source)
Step #0 - "mavenBuild":    at com.google.api.gax.grpc.ChannelPool.<init>(ChannelPool.java:107)
Step #0 - "mavenBuild":    at com.google.api.gax.grpc.ChannelPool.create(ChannelPool.java:85)
Step #0 - "mavenBuild":    at com.google.api.gax.grpc.InstantiatingGrpcChannelProvider.createChannel(InstantiatingGrpcChannelProvider.java:237)
Step #0 - "mavenBuild":    at com.google.api.gax.grpc.InstantiatingGrpcChannelProvider.getTransportChannel(InstantiatingGrpcChannelProvider.java:231)
Step #0 - "mavenBuild":    at com.google.api.gax.rpc.ClientContext.create(ClientContext.java:236)
Step #0 - "mavenBuild":    at com.google.api.gax.rpc.ClientContext.create(ClientContext.java:143)
Step #0 - "mavenBuild":    at com.google.cloud.logging.spi.v2.GrpcLoggingRpc.<init>(GrpcLoggingRpc.java:150)
Step #0 - "mavenBuild":    at com.google.cloud.logging.LoggingOptions$DefaultLoggingRpcFactory.create(LoggingOptions.java:66)
Step #0 - "mavenBuild":    at com.google.cloud.logging.LoggingOptions$DefaultLoggingRpcFactory.create(LoggingOptions.java:60)
Step #0 - "mavenBuild":    at com.google.cloud.ServiceOptions.getRpc(ServiceOptions.java:564)
Step #0 - "mavenBuild":    at com.google.cloud.storage.HttpStorageOptions.getStorageRpcV1(HttpStorageOptions.java:78)
Step #0 - "mavenBuild":    at com.google.cloud.storage.StorageImpl.<init>(StorageImpl.java:117)
Step #0 - "mavenBuild":    at com.google.cloud.storage.HttpStorageOptions$HttpStorageFactory.create(HttpStorageOptions.java:283)
Step #0 - "mavenBuild":    at com.google.cloud.storage.HttpStorageOptions$HttpStorageFactory.create(HttpStorageOptions.java:257)
Step #0 - "mavenBuild":    at com.google.cloud.ServiceOptions.getService(ServiceOptions.java:544)
Step #0 - "mavenBuild":    at io.quarkiverse.googlecloudservices.logging.runtime.cdi.LoggingProducer.create(LoggingProducer.java:43)
Step #0 - "mavenBuild":    at io.quarkiverse.googlecloudservices.logging.runtime.cdi.LoggingProducer_ProducerMethod_create_43cdc485b040e9e622e14fa50960c6f67837b748_Bean.doCreate(Unknown Source)
Step #0 - "mavenBuild":    at io.quarkiverse.googlecloudservices.logging.runtime.cdi.LoggingProducer_ProducerMethod_create_43cdc485b040e9e622e14fa50960c6f67837b748_Bean.create(Unknown Source)
Step #0 - "mavenBuild":    at io.quarkiverse.googlecloudservices.logging.runtime.cdi.LoggingProducer_ProducerMethod_create_43cdc485b040e9e622e14fa50960c6f67837b748_Bean.create(Unknown Source)
Step #0 - "mavenBuild":    at io.quarkus.arc.impl.DependentContext.get(DependentContext.java:31)
Step #0 - "mavenBuild":    at io.quarkus.resteasy.reactive.server.runtime.exceptionmappers.UnauthorizedExceptionMapper$GeneratedExceptionHandlerFor$UnauthorizedException$OfMethod$handle_Bean.get(Unknown Source)
Step #0 - "mavenBuild":    at io.quarkus.resteasy.reactive.server.runtime.exceptionmappers.UnauthorizedExceptionMapper$GeneratedExceptionHandlerFor$UnauthorizedException$OfMethod$handle_Bean.get(Unknown Source)
Step #0 - "mavenBuild":    at io.quarkus.arc.impl.InstanceImpl.getBeanInstance(InstanceImpl.java:325)
Step #0 - "mavenBuild":    at io.quarkus.arc.impl.InstanceImpl.getInternal(InstanceImpl.java:309)
Step #0 - "mavenBuild":    at io.quarkus.arc.impl.InstanceImpl$$Lambda$1040/0x00000007c278cad0.get(Unknown Source)
Step #0 - "mavenBuild":    at io.quarkus.arc.impl.ArcContainerImpl$2.get(ArcContainerImpl.java:184)
Step #0 - "mavenBuild":    at io.quarkus.arc.impl.ArcContainerImpl$2.get(ArcContainerImpl.java:179)
Step #0 - "mavenBuild":    at io.quarkus.arc.impl.LazyValue.get(LazyValue.java:32)
Step #0 - "mavenBuild":    at io.quarkus.arc.impl.ArcContainerImpl.getMatchingRemovedBeans(ArcContainerImpl.java:744)
Step #0 - "mavenBuild":    at io.quarkus.arc.impl.ArcContainerImpl.scanRemovedBeans(ArcContainerImpl.java:758)
Step #0 - "mavenBuild":    at io.quarkus.arc.impl.ArcContainerImpl.getBean(ArcContainerImpl.java:565)
Step #0 - "mavenBuild":    at io.quarkus.arc.impl.ArcContainerImpl.instanceHandle(ArcContainerImpl.java:514)
Step #0 - "mavenBuild":    at io.quarkus.arc.impl.ArcContainerImpl.instance(ArcContainerImpl.java:279)
Step #0 - "mavenBuild":    at io.quarkiverse.googlecloudservices.logging.runtime.LoggingHandler.initLogger(LoggingHandler.java:160)
Step #0 - "mavenBuild":    at io.quarkiverse.googlecloudservices.logging.runtime.LoggingHandler.initGetLogging(LoggingHandler.java:110)
Step #0 - "mavenBuild":    at io.quarkiverse.googlecloudservices.logging.runtime.LoggingHandler.flush(LoggingHandler.java:101)
Step #0 - "mavenBuild":    at org.jboss.logmanager.ExtHandler.flush(ExtHandler.java:324)
Step #0 - "mavenBuild":    at org.jboss.logmanager.ExtHandler.doPublish(ExtHandler.java:140)
Step #0 - "mavenBuild":    at io.quarkus.bootstrap.logging.QuarkusDelayedHandler.doPublish(QuarkusDelayedHandler.java:88)
Step #0 - "mavenBuild":    at org.jboss.logmanager.ExtHandler.publish(ExtHandler.java:88)
Step #0 - "mavenBuild":    at org.jboss.logmanager.LoggerNode.publish(LoggerNode.java:434)
Step #0 - "mavenBuild":    at org.jboss.logmanager.Logger.logRaw(Logger.java:1089)
Step #0 - "mavenBuild":    at org.jboss.logmanager.Logger.log(Logger.java:1052)
Step #0 - "mavenBuild":    at org.jboss.logging.JBossLogManagerLogger.doLog(JBossLogManagerLogger.java:45)
Step #0 - "mavenBuild":    at org.jboss.logging.Logger.error(Logger.java:1507)
Step #0 - "mavenBuild":    at io.quarkus.runtime.Quarkus.run(Quarkus.java:79)
Step #0 - "mavenBuild":    at io.quarkus.runtime.Quarkus.run(Quarkus.java:44)
Step #0 - "mavenBuild": 
Step #0 - "mavenBuild": 	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.util.AnalysisError.parsingError(AnalysisError.java:153)
Step #0 - "mavenBuild": 	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.flow.MethodTypeFlow.createFlowsGraph(MethodTypeFlow.java:104)
Step #0 - "mavenBuild": 	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.flow.MethodTypeFlow.ensureFlowsGraphCreated(MethodTypeFlow.java:83)
Step #0 - "mavenBuild": 	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.flow.MethodTypeFlow.getOrCreateMethodFlowsGraph(MethodTypeFlow.java:65)
Step #0 - "mavenBuild": 	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.typestate.DefaultVirtualInvokeTypeFlow.onObservedUpdate(DefaultVirtualInvokeTypeFlow.java:109)
Step #0 - "mavenBuild": 	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.flow.TypeFlow.update(TypeFlow.java:562)
Step #0 - "mavenBuild": 	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.PointsToAnalysis$1.run(PointsToAnalysis.java:488)
Step #0 - "mavenBuild": 	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.util.CompletionExecutor.executeCommand(CompletionExecutor.java:193)
Step #0 - "mavenBuild": 	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.util.CompletionExecutor.lambda$executeService$0(CompletionExecutor.java:177)
Step #0 - "mavenBuild": 	at java.base/java.util.concurrent.ForkJoinTask$RunnableExecuteAction.exec(ForkJoinTask.java:1395)
Step #0 - "mavenBuild": 	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373)
Step #0 - "mavenBuild": 	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1182)
Step #0 - "mavenBuild": 	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1655)
Step #0 - "mavenBuild": 	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1622)
Step #0 - "mavenBuild": 	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165)
Step #0 - "mavenBuild": Caused by: org.graalvm.compiler.java.BytecodeParser$BytecodeParserError: com.oracle.graal.pointsto.constraints.UnresolvedElementException: Discovered unresolved method during parsing: io.grpc.alts.GoogleDefaultChannelCredentials$Builder.callCredentials(io.grpc.CallCredentials). This error is reported at image build time because class com.google.api.gax.grpc.InstantiatingGrpcChannelProvider is registered for linking at image build time by command line
Step #0 - "mavenBuild": 	at parsing com.google.api.gax.grpc.InstantiatingGrpcChannelProvider.createSingleChannel(InstantiatingGrpcChannelProvider.java:328)
Step #0 - "mavenBuild": 	at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.throwParserError(BytecodeParser.java:2518)
Step #0 - "mavenBuild": 	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.phases.SharedGraphBuilderPhase$SharedBytecodeParser.throwParserError(SharedGraphBuilderPhase.java:110)
Step #0 - "mavenBuild": 	at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.iterateBytecodesForBlock(BytecodeParser.java:3393)
Step #0 - "mavenBuild": 	at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.handleBytecodeBlock(BytecodeParser.java:3345)
Step #0 - "mavenBuild": 	at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.processBlock(BytecodeParser.java:3190)
Step #0 - "mavenBuild": 	at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.build(BytecodeParser.java:1138)
Step #0 - "mavenBuild": 	at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.buildRootMethod(BytecodeParser.java:1030)
Step #0 - "mavenBuild": 	at jdk.internal.vm.compiler/org.graalvm.compiler.java.GraphBuilderPhase$Instance.run(GraphBuilderPhase.java:97)
Step #0 - "mavenBuild": 	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.phases.SharedGraphBuilderPhase.run(SharedGraphBuilderPhase.java:84)
Step #0 - "mavenBuild": 	at jdk.internal.vm.compiler/org.graalvm.compiler.phases.Phase.run(Phase.java:49)
Step #0 - "mavenBuild": 	at jdk.internal.vm.compiler/org.graalvm.compiler.phases.BasePhase.apply(BasePhase.java:446)
Step #0 - "mavenBuild": 	at jdk.internal.vm.compiler/org.graalvm.compiler.phases.Phase.apply(Phase.java:42)
Step #0 - "mavenBuild": 	at jdk.internal.vm.compiler/org.graalvm.compiler.phases.Phase.apply(Phase.java:38)
Step #0 - "mavenBuild": 	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.flow.AnalysisParsedGraph.parseBytecode(AnalysisParsedGraph.java:135)
Step #0 - "mavenBuild": 	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.meta.AnalysisMethod.ensureGraphParsed(AnalysisMethod.java:685)
Step #0 - "mavenBuild": 	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.phases.InlineBeforeAnalysisGraphDecoder.lookupEncodedGraph(InlineBeforeAnalysis.java:180)
Step #0 - "mavenBuild": 	at jdk.internal.vm.compiler/org.graalvm.compiler.replacements.PEGraphDecoder.doInline(PEGraphDecoder.java:1162)
Step #0 - "mavenBuild": 	at jdk.internal.vm.compiler/org.graalvm.compiler.replacements.PEGraphDecoder.tryInline(PEGraphDecoder.java:1145)
Step #0 - "mavenBuild": 	at jdk.internal.vm.compiler/org.graalvm.compiler.replacements.PEGraphDecoder.trySimplifyInvoke(PEGraphDecoder.java:1003)
Step #0 - "mavenBuild": 	at jdk.internal.vm.compiler/org.graalvm.compiler.replacements.PEGraphDecoder.handleInvoke(PEGraphDecoder.java:957)
Step #0 - "mavenBuild": 	at jdk.internal.vm.compiler/org.graalvm.compiler.nodes.GraphDecoder.processNextNode(GraphDecoder.java:817)
Step #0 - "mavenBuild": 	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.phases.InlineBeforeAnalysisGraphDecoder.processNextNode(InlineBeforeAnalysis.java:240)
Step #0 - "mavenBuild": 	at jdk.internal.vm.compiler/org.graalvm.compiler.nodes.GraphDecoder.decode(GraphDecoder.java:548)
Step #0 - "mavenBuild": 	at jdk.internal.vm.compiler/org.graalvm.compiler.replacements.PEGraphDecoder.decode(PEGraphDecoder.java:833)
Step #0 - "mavenBuild": 	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.phases.InlineBeforeAnalysis.decodeGraph(InlineBeforeAnalysis.java:98)
Step #0 - "mavenBuild": 	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.flow.MethodTypeFlowBuilder.parse(MethodTypeFlowBuilder.java:179)
Step #0 - "mavenBuild": 	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.flow.MethodTypeFlowBuilder.apply(MethodTypeFlowBuilder.java:349)
Step #0 - "mavenBuild": 	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.flow.MethodTypeFlow.createFlowsGraph(MethodTypeFlow.java:93)
Step #0 - "mavenBuild": 	... 13 more
Step #0 - "mavenBuild": Caused by: com.oracle.graal.pointsto.constraints.UnresolvedElementException: Discovered unresolved method during parsing: io.grpc.alts.GoogleDefaultChannelCredentials$Builder.callCredentials(io.grpc.CallCredentials). This error is reported at image build time because class com.google.api.gax.grpc.InstantiatingGrpcChannelProvider is registered for linking at image build time by command line
Step #0 - "mavenBuild": 	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.phases.SharedGraphBuilderPhase$SharedBytecodeParser.reportUnresolvedElement(SharedGraphBuilderPhase.java:333)
Step #0 - "mavenBuild": 	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.phases.SharedGraphBuilderPhase$SharedBytecodeParser.handleUnresolvedMethod(SharedGraphBuilderPhase.java:323)
Step #0 - "mavenBuild": 	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.phases.SharedGraphBuilderPhase$SharedBytecodeParser.handleUnresolvedInvoke(SharedGraphBuilderPhase.java:279)
Step #0 - "mavenBuild": 	at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.genInvokeVirtual(BytecodeParser.java:1721)
Step #0 - "mavenBuild": 	at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.processBytecode(BytecodeParser.java:5286)
Step #0 - "mavenBuild": 	at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.iterateBytecodesForBlock(BytecodeParser.java:3385)
Step #0 - "mavenBuild": 	... 38 more
Step #0 - "mavenBuild": ------------------------------------------------------------------------------------------------------------------------
Step #0 - "mavenBuild":                         12.9s (7.1% of total time) in 25 GCs | Peak RSS: 3.54GB | CPU load: 1.93
Step #0 - "mavenBuild": ========================================================================================================================
Step #0 - "mavenBuild": Failed generating 'app-0.1.0-runner' after 2m 59s.
Step #0 - "mavenBuild": Error: Image build request failed with exit status 1
Step #0 - "mavenBuild": [INFO] ------------------------------------------------------------------------
Step #0 - "mavenBuild": [INFO] BUILD FAILURE
Step #0 - "mavenBuild": [INFO] ------------------------------------------------------------------------
Step #0 - "mavenBuild": [INFO] Total time:  05:46 min
Step #0 - "mavenBuild": [INFO] Finished at: 2023-11-09T21:26:19Z
Step #0 - "mavenBuild": [INFO] ------------------------------------------------------------------------
Step #0 - "mavenBuild": [ERROR] Failed to execute goal io.quarkus.platform:quarkus-maven-plugin:3.5.1:build (default) on project app: Failed to build quarkus application: io.quarkus.builder.BuildException: Build failure: Build failed due to errors
Step #0 - "mavenBuild": [ERROR] 	[error]: Build step io.quarkus.deployment.pkg.steps.NativeImageBuildStep#build threw an exception: io.quarkus.deployment.pkg.steps.NativeImageBuildStep$ImageGenerationFailureException: Image generation failed. Exit code: 1
Step #0 - "mavenBuild": [ERROR] 	at io.quarkus.deployment.pkg.steps.NativeImageBuildStep.imageGenerationFailed(NativeImageBuildStep.java:474)
Step #0 - "mavenBuild": [ERROR] 	at io.quarkus.deployment.pkg.steps.NativeImageBuildStep.build(NativeImageBuildStep.java:264)
Step #0 - "mavenBuild": [ERROR] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
Step #0 - "mavenBuild": [ERROR] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
Step #0 - "mavenBuild": [ERROR] 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
Step #0 - "mavenBuild": [ERROR] 	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
Step #0 - "mavenBuild": [ERROR] 	at io.quarkus.deployment.ExtensionLoader$3.execute(ExtensionLoader.java:849)
Step #0 - "mavenBuild": [ERROR] 	at io.quarkus.builder.BuildContext.run(BuildContext.java:282)
Step #0 - "mavenBuild": [ERROR] 	at org.jboss.threads.ContextHandler$1.runWith(ContextHandler.java:18)
Step #0 - "mavenBuild": [ERROR] 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2513)
Step #0 - "mavenBuild": [ERROR] 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1538)
Step #0 - "mavenBuild": [ERROR] 	at java.base/java.lang.Thread.run(Thread.java:833)
Step #0 - "mavenBuild": [ERROR] 	at org.jboss.threads.JBossThread.run(JBossThread.java:501)
Step #0 - "mavenBuild": [ERROR] -> [Help 1]
Step #0 - "mavenBuild": [ERROR] 
Step #0 - "mavenBuild": [ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
Step #0 - "mavenBuild": [ERROR] Re-run Maven using the -X switch to enable full debug logging.
Step #0 - "mavenBuild": [ERROR] 
Step #0 - "mavenBuild": [ERROR] For more information about the errors and possible solutions, please read the following articles:
Step #0 - "mavenBuild": [ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
```
How can we proceed to fix this error ?

Thank you all

--- a/logging/runtime/pom.xml
+++ b/logging/runtime/pom.xml
@@ -18,12 +18,8 @@
             <artifactId>quarkus-google-cloud-common</artifactId>
         </dependency>
         <dependency>
-            <groupId>io.quarkus</groupId>
-            <artifactId>quarkus-grpc-common</artifactId>
-        </dependency>
-        <dependency>
-            <groupId>com.google.cloud</groupId>
-            <artifactId>google-cloud-core</artifactId>
+            <groupId>io.quarkiverse.googlecloudservices</groupId>
+            <artifactId>quarkus-google-cloud-common-grpc</artifactId>
         </dependency>
         <dependency>
             <groupId>com.google.cloud</groupId>
@@ -50,6 +46,10 @@
                     <artifactId>animal-sniffer-annotations</artifactId>
                 </exclusion>
             </exclusions>
+        </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-grpc-common</artifactId>
         </dependency>
 
         <!-- Tests -->

